src/item/StringToItemParser.php

```diff
@@ -615,6 +615,7 @@ private static function make() : self{
 		$result->registerBlock("jungle_wood", fn() => VanillaBlocks::JUNGLE_WOOD());
 		$result->registerBlock("jungle_wood_stairs", fn() => VanillaBlocks::JUNGLE_STAIRS());
 		$result->registerBlock("jungle_wooden_stairs", fn() => VanillaBlocks::JUNGLE_STAIRS());
+		$result->registerBlock("kelp_block", fn() => VanillaBlocks::KELP());
```

Comment:
StringToItemParser should not include blocks that can't be obtained in item form.